sdk/identity/Azure.Identity/src/EnvironmentVariables.cs

```diff
@@ -13,6 +13,7 @@ internal class EnvironmentVariables
         public static string ClientId => Environment.GetEnvironmentVariable("AZURE_CLIENT_ID");
         public static string ClientSecret => Environment.GetEnvironmentVariable("AZURE_CLIENT_SECRET");
         public static string ClientCertificatePath => Environment.GetEnvironmentVariable("AZURE_CLIENT_CERTIFICATE_PATH");
+        public static string ClientCertificateEnableSni => Environment.GetEnvironmentVariable("AZURE_CLIENT_CERTIFICATE_ENABLE_SNI");
```

Comment:
Discussed this with the architecture board. We'd like to change this environment variable name to `AZURE_CLIENT_SEND_CERTIFICATE_CHAIN`